mongo/votes.go

```diff
@@ -81,6 +81,37 @@ func (ms *MongoStorage) VotersOfElection(electionID types.HexBytes) ([]*User, er
 	return users, nil
 }
 
+// HasAlreadyVoted returns true if the user has already voted in the election.
+// If the user is not a participant of the election, it returns an error. If
+// the user has already voted, it returns true, otherwise false. If something
+// goes wrong, it returns an error.
+func (ms *MongoStorage) HasAlreadyVoted(userFID uint64, electionID types.HexBytes) (bool, error) {
```

Comment:
This query performs three mongoDB queries, looks a bit overkill to me. What's the reason? Why not just to check if a voter voted an election using "voters" table? Is it really needed to check the census and previously the user?

Reply:
I try to make sure that the user is included in the census, but you are right, if the user has already voted, he is also included in the census.

Reply:
I think about it and I think that the check is necessary because this method is used to check if a user can be notified (they have not yet voted but they are in the census).